pkg/rpm/resolver/templates/rpm-resolution.sh.tpl

```diff
@@ -10,9 +10,13 @@ set -euo pipefail
 #  LocalGPGList - list of local GPG keys that will be imported in the resolver image
 #  NoGPGCheck   - when set to true skips the GPG validation for all third-party repositories and local RPMs
 #  Arch         - sets the architecture of the rpm packages to pull
+#  EnableExtras - registers the SL-Micro-Extras repo for use in resolution
 
 {{ if ne .RegCode "" }}
 suseconnect -r {{ .RegCode }}
+{{ if $.EnableExtras -}}
```

Comment:
Nit: The dollar sign shouldn't be necessary.

Reply:
Admittedly, I didn't even pull up the template syntax docs. I looked at the GPG conditional a few lines below and followed that model. Still, I appreciate the heads up.